adapters/tlsio_openssl.c

```diff
@@ -880,14 +887,448 @@ static int load_system_store(TLS_IO_INSTANCE* tls_io_instance)
             X509_free(x509);
         }
     }    
+
+    // load all the revocation lists into the openSSL crl store
+    PCCRL_CONTEXT pCRLContext = NULL;
+    while (1)
+    {
+        /* To free a context obtained by a find or enumerate function,
+         * either pass it in as the previous context parameter to a
+         * subsequent invocation of the function, or call the appropriate
+         * free function. --from MSDN
+        */
+        pCRLContext = CertEnumCRLsInStore(hSysStore, pCRLContext);
+        if (!pCRLContext)
+        {
+            break;
+        }
+
+        const unsigned char *encoded_crl = pCRLContext->pbCrlEncoded;
+
+        X509_CRL *x509_crl = d2i_X509_CRL(NULL, &encoded_crl, pCRLContext->cbCrlEncoded);
+        if (x509_crl)
+        {
+            int i = X509_STORE_add_crl(store, x509_crl);
+            if (i != 1)
+            {
+                LogError("revocation list adding failed.");
+            }
+            X509_CRL_free(x509_crl);
+        }
+    }
+
+    // setup CRL checking
```

Comment:
@wolfma61, @mahilleb-msft 
we have a problem here,
since we are using ssl, not curl on uwp, i cannot download the crl (since the builtin protocol is only https, not http.)

therefore, i disable the store flags for now.
we need a discussion how to proceed.

btw, different ssl versions for android and uwp does not make this easier...